### ISSUE TITLE
Entity states are not updated

### ISSUE THREAD

**Issue by Eskander** [vermut/ha_amc_alarm#5]
Reloading the integration adds the correct states to entities, but subsequent state changes are not reflected.

Central: X64V/4.20 (amcProtoVer 2)
Logs: [home-assistant_amc_alarm_2024-06-10T13-41-28.729Z.log.txt](https://github.com/user-attachments/files/15774277/home-assistant_amc_alarm_2024-06-10T13-41-28.729Z.log.txt)

I previously reported in #4 that the latest integration update worked correctly when it successfully logged in, but did not pay attention to state updates.

**Comment by GitWally** [comment]
@vermut I am un the very same situazione.
I have a X864/4.10 but, as far as I can see entity status is updated only upon integration reload.

So question is: is it an intended behaviour or a bug?
It would be helpful to get updated also to create presence sensors.
Thanks

**Comment by vermut** [comment]
This is a bug, and I need to go and implement a fix. Seems like in new versions they are finally sending only changed fields, not the whole huge state.

**Comment by vermut** [comment]
@Eskander  @GitWally  try version v1.1.0

**Comment by Eskander** [comment]
Yep, entities are updated correctly with version v1.1.0, thanks for working on this.

**Comment by GitWally** [comment]
> @Eskander  @GitWally  try version v1.1.0

Dowloaded and testing!
Seems to do the job: it could be very usefull to me for presence sensor and to understand when door are opening / closing!

Great @vermut !

**Comment by vermut** [comment]
Merged downstream fix.
